docs/variants/qemu_q35/test-matrix.md

```diff
@@ -20,7 +20,6 @@ Procedure for tested platform: `QEMU Q35 Machine`
 | 6.   | [Windows booting][WBT]                            | WBT           | All                                  |
 | 7.   | [Qubes OS Support][QBS]                           | QBS           | QBS001.001, QBS001.002               |
 | 8.   | [CPU status][CPU]                                 | CPU           | All                                  |
-| 9.   | [BIOS Setup password][PSW]                        | PSW           | All                                  |
```

Comment:
@Asiderr removed?

Reply:
@artur-rs this page doesn't exist

Reply:
@Asiderr @pietrushnic I found this page. Name was changed twice, now it is correct. 9fd2cb275c8c459ff93f1c551b08086ffb0ba1bd